src/scss/components/newsletter-form/newsletter-form.stories.ts

```diff
@@ -0,0 +1,36 @@
+import { html } from "lit";
+import type { Meta, StoryObj } from "@storybook/web-components";
+
+const meta: Meta = {
+  title: "Components/Newsletter Form",
+  parameters: {
+    backgrounds: {
+      default: 'dark',
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj;
+
+export const Default: Story = {
+  render: () => html`
+    <form class="iati-newsletter-form">
+      <div class="iati-newsletter-form__item">
+        <label for="newsletter-email" class="iati-newsletter-form__label">Email</label>
+        <input id="newsletter-email" type="text" class="iati-newsletter-form__input" />
```

Comment:
The outline on this inputs is adding some padding and causing them to jump around a little when clicked between

Reply:
@tillywoodfield can you confirm the OS\browser you were using for this test please.

Reply:
I'm using macOS and Chrome 129

Reply:
@tillywoodfield I'm not sure if I'm able to reproduce what you've described above. When I focus the inputs I get the outline but no change in layout...

![bs_macsqa_Chrome_129 0 beta](https://github.com/user-attachments/assets/d8c4e160-61b1-49ba-972a-977b222130e2)